pkg/kubelet/events/event.go

```diff
@@ -85,8 +85,10 @@ const (
 
 // Probe event reason list
 const (
-	ContainerUnhealthy    = "Unhealthy"
-	ContainerProbeWarning = "ProbeWarning"
+	ContainerUnhealthy          = "Unhealthy"
+	ContainerProbeWarning       = "ProbeWarning"
+	ContainerProbeResultFailure = "ContainerProbeResultFailure"
+	ContainerProbeResultSuccess = "ContainerProbeResultSuccess"
```

Comment:
Would it make sense to make new events consistent with the existing ones?
```suggestion
	ContainerProbeResultFailure = "ProbeResultFailure"
	ContainerProbeResultSuccess = "ProbeResultSuccess"
```